gpuarray/src/private_config.h

```diff
@@ -0,0 +1,53 @@
+/******************************************************************
```

Comment:
The comments on the `#else` and `#endif` are sometimes useful, but sometimes just annoying because the matching `#if` is just two lines above.  In the cases where there is significant separation between statements, a comment is useful, but in most cases you fixed, it is not.